index.html

```diff
@@ -366,14 +380,27 @@ <h2><dfn>PerformanceObserverInit</dfn> dictionary</h2>
           <pre class="idl">
           dictionary PerformanceObserverInit {
             required sequence&lt;DOMString&gt; entryTypes;
-            boolean buffered = false;
           };
           </pre>
           <dl>
             <dt><dfn>entryTypes</dfn></dt>
             <dd>A list of entry types to be observed. The list MUST NOT be empty
             and types not recognized by the user agent MUST be ignored.</dd>
           </dl>
+          <p class="note">Whenever a new kind of <a>PerformanceEntry</a> is defined,
+            the <a>PerformanceObserverInit</a> dictionary should be augmented to
+            include a new optional <a>PerformanceEntryObserverOptions</a> dictionary
+            with the name of the new <a data-lt="PerformanceEntry.entryType">
```

Comment:
It's not clear what you mean by "the name of the new entryType". Can you be more explicit?

Reply:
What I mean is that `PerformanceObserverInit` needs to be augmented for example if we were adding EventTiming with entries that have entryType 'event':
```
partial dictionary PerformanceObserverInit {
   optional PerformanceEntryObserverOptions event;
}
```
I reworded that last part, is it better now?